docs/fast_data/monitoring/overview.md

```diff
@@ -6,38 +6,131 @@ sidebar_label: Overview
-
-There are three Dashboards ready to use:
-
-1. [Consumer Groups](/fast_data/monitoring/dashboards/consumer_groups.md)
-2. [Projection Changes](/fast_data/monitoring/dashboards/projection_changes.md)
-3. [Single Views](/fast_data/monitoring/dashboards/single_views.md)
-
-More details on what information they provide in their specific pages.
-
-## Putting It All Together
-
-Once your Dashboards are ready, it can be very handy to view them directly inside the console. To do that, you can visit the `Dashboards` section of your project and add the Dashboards you just created to it, as explained [here](/development_suite/monitoring/dashboard.md#add-a-dashboard).
+The Grafana Dashboards make use of [Prometheus](https://prometheus.io/) metrics, which means that you should enable the monitoring of your project (for example, using [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator)) and export the metrics of interest.
+
+If you are planning to use Prometheus Operator as a monitoring system for your project, you can automatically generate and configure the `servicemonitor.yml` files for all the monitoring services using their [Metrics card](/development_suite/api-console/api-design/microservice-monitoring.md).
+
+## Metrics
+
+Metrics are a set of categorized data generally exposed through a `/-/metrics` http GET route which reveals all the Prometheus metrics exposed by each service.
+
+### Fast Data services metrics
+
+Each Fast Data service comes with its set of metrics exposed through an http route. To expose them you only need to enable the `Enable metrics gathering` switch that you can find in each microservice's page in the console.
+
+![Enable metrics toggle in microservice page from the console](../img/monitoring-enable-metrics-toggle.png)
+
+### MongoDB and Apache Kafka metrics
+
+For the dashboards related to MongoDB and Apache Kafka metrics you will need to setup the [Kafka Exporter](https://github.com/danielqsj/kafka_exporter) and [MongoDB Query Exporter](https://github.com/raffis/mongodb-query-exporter) services. Conveniently, you can create them all at once by going to the Applications section of your console and creating a new `Fast Data Monitoring` Application.
+
+#### MongoDB Query Exporter Configuration
+
+The MongoDB Query Exporter service will expose all metrics we define in the `config.yml` config map. To use the Projection Changes Dashboard you need to make sure that for every Projection Changes collection you have in the console the respective metric is defined.
+
+Here's an example with `fd-pc-library` and `fd-pc-restaurant` Projection Changes collections:
+
+```yml title="config.yml"
+...
+- name: projection_changes_new_info
+  type: counter #Can also be empty, the default is gauge
+  servers: [main] #Can also be empty, if empty the metric will be used for every server defined
+  help: 'Count the number of NEW projection changes'
+  value: total
+  labels: []
```

Comment:
Honestly I didn't know we could do that. Where is that we can provide a default for the `mongodb-prometheus-exporter`?

Reply:
A default is already provided in the plugin, when creating the Fast Data Monitoring application. However, the idea is to update that config and reflect that one the documentation.

Reply:
Currently we are not deploying the latest version of the `mongodb-prometheus-exporter`. We should upgrade it and adopt the [v3 configuration](https://github.com/raffis/mongodb-query-exporter#configure-metrics), where for each metric it is possible to configure the property `emptyValue` that provides a "default" value when the metrics does not return any.

However, this might be done in another PR, so for the moment we may close this discussion.